curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc98cfe781112ace7d3ee.md

```diff
@@ -0,0 +1,44 @@
+---
+id: 656cc98cfe781112ace7d3ee
+title: Task 107
+challengeType: 22
+dashedName: task-107
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Sophie: Today, I'm going out. I can show you some places around here. Are you interested?
+-->
+
+# --description--
+
```

Comment:
@nieldakarla, this fill in the blank exercise has no description. Should it be like this? Suggestion: "Now let's use the expression we learned for referring to nearby places."